.gitleaks.toml

```diff
@@ -195,6 +195,7 @@ files = [
   "test/ssh/id_(.*)", # since-removed ssh test certs
   "test/util/ssl/(.*)", # test ssl certs
   "internal/plugin/connectors/tcp/mssql/connection_details_test.go", # fake cert string
+  "test/connector/http/conjur/docker-compose.yml",  # fake password
```

Comment:
would it be better to have this as a regex as `SuperSecure`?

adding the file exception leaves open the possibility that at some point in the future we'll add a real secret to this file and gitleaks will ignore it